go/mysql/auth_server_static.go

```diff
@@ -153,6 +155,19 @@ func (a *AuthServerStatic) installSignalHandlers() {
 			a.loadConfigFromParams(*mysqlAuthServerStaticFile, "")
 		}
 	}()
+
+	// If duration is set, it will reload configuration every interval
+	if *staticFileAuthTickerDuration > 0 {
+		ticker := time.NewTicker(*staticFileAuthTickerDuration)
+		go func() {
+			for {
+				select {
+				case <-ticker.C:
+					signal.Notify(sigChan, syscall.SIGHUP)
```

Comment:
This call only registers sigChan to receive a notification on SIGHUP.
Instead, you need to send a notification:

```suggestion
					sigChan <- syscall.SIGHUP
```

Reply:
Resolved